Yosemite/Yosemite/Actions/BlazeAction.swift

```diff
@@ -17,6 +17,20 @@ public enum BlazeAction: Action {
                         siteID: Int64,
                         onCompletion: (Result<Void, Error>) -> Void)
 
+    /// Retrieves and stores Brief Blaze Campaign Info for a site
```

Comment:
Nit:

```suggestion
    /// Retrieves and stores Blaze Campaign items for a site using v1.1 endpoint.
```

Reply:
Good catch. Done in https://github.com/woocommerce/woocommerce-ios/pull/12127/commits/c3122a06cf09fe9b464be13add67b845069d4db0